src/specific/s_misc.c

```diff
@@ -9,26 +9,6 @@
 
 #include <string.h>
 
-void S_FadeInInventory(int32_t fade)
```

Comment:
Is it possible to add in fade to T1M? Would be nice for the pause and inventory screens. It's not a huge deal either, but with this refactor, would it be possible to add in fading? There's a noticeable black screen between the level ending stats screen and the start of the next level. I guess an option load screen return could also mask this.

According to Arsunt:
```
Can new T1M picture implementation make picture darker? 
To draw picture with alpha?

The best approach (used in T2M inventory darkened inventory style) is non blocking fade out, so player don't need to wait fade then inventory opening animation. Both happens in the same time. So simply while you render inventory normally, you continue to darken background down to 50% brightness.
You update background every frame anyway. So everytime you do it you can change brightness from current value towards goal value, using some fade speed.
Just set somewhere out, current value, goal value, and speed (how many milliseconds you need to fade from 100 to 0)

For you it is simpler to draw fullscreen black rectangle with some alpha (255-alpha) over picture
I mean you draw picture normally, then draw rectangle (if alpha is not 255)
```

Maybe that should be a separate merge request though.

Reply:
Thanks. Like you said, I don't think this is related to this PR. LMK if the black frame is a regression I introduced here, or something that we already had happen.

https://github.com/rr-/Tomb1Main/discussions/333